examples/jsm/helpers/LightProbeHelper.js

```diff
@@ -112,9 +94,7 @@ function LightProbeHelper( lightProbe, size ) {
 
 			'	vec3 outgoingLight = RECIPROCAL_PI * irradiance * intensity;',
 
-			'	outgoingLight = linearToOutput( outgoingLight );',
-
-			'	gl_FragColor = vec4( outgoingLight, 1.0 );',
+			'	gl_FragColor = linearToOutputTexel( vec4( outgoingLight, 1.0 ) );',
```

Comment:
Why not including the `encodings_fragment` here?

Reply:
My preference. I did not include `common.glsl` either. I only included the code I needed to keep this shader simple.

Reply:
Okay, I see. I leave this detail up to @mrdoob.